docs/conf.py

```diff
@@ -345,10 +345,13 @@
 
 # Example configuration for intersphinx: refer to the Python standard library.
 intersphinx_mapping = {
-    "python": ("http://python.readthedocs.org/en/latest/", None),
-    "google-auth": ("https://google-auth.readthedocs.io/en/stable", None),
+    "python": ("https://python.readthedocs.org/en/latest/", None),
+    "google-auth": (
+        "https://googleapis.dev/python/google-auth/latest/index.html",
```

Comment:
Need to regenerate to remove `/index.html`?

Reply:
Yep. I'll just close this and let autosynth open an new one tomorrow.